vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -148,6 +156,10 @@ export const FormInput = {
         footerMinWidth: {
             type: Number,
             default: null
+        },
+        tooltipProps: {
+            type: Object,
+            default: null

```

Comment:
Isn't this logging warnings? `Object` props needs a factory function. Reference: https://vuejs.org/guide/components/props.html#prop-validation

It should be:

```
tooltipProps: {
   type: Object,
   default: () => ({})
}
```

Reply:
No warnings were logged, but I wasn't aware of this! I'll change it up